packages/two_dimensional_scrollables/lib/src/table_view/table_span.dart

```diff
@@ -296,12 +296,19 @@ class TableSpanDecoration {
   const TableSpanDecoration({
     this.border,
     this.color,
+    this.borderRadius,
     this.consumeSpanPadding = true,
   });
 
   /// The border drawn around the span.
   final TableSpanBorder? border;
 
+  /// The radius by which the leading and trailing ends of a row or
+  /// column will be rounded.
+  ///
+  /// Applies to the [border] and [color] of the given [TableSpan].
+  final BorderRadius? borderRadius;
```

Comment:
For my own understanding: Why is this a property on TableSpanDecoration instead of on TableSpanBorder? Just going by the name, it seems to be a border property?

Reply:
Because it applies to both the color and the border. I modeled this to follow BoxDecoration, which is the same.

Reply:
👍